### DISCUSSION TITLE
Cannot get Windows authentication to work

### DISCUSSION THREAD

**Discussion by ajbrehm** [Badgerati/Pode#1298] [General]
I am following the instructions on https://badgerati.github.io/Pode/Tutorials/Authentication/Inbuilt/WindowsLocal/.

But apparently I am not clever enough, because adding

    New-PodeAuthScheme -Basic | Add-PodeAuthWindowsLocal -Name 'Login' -Sessionless

    Add-PodeAuthMiddleware -Name 'GlobalAuthValidation' -Authentication 'Login'

to my Start-PodeServer always gives me 

InvalidOperation: [...]\PowerShell\Modules\Pode\2.10.0\Private\Authentication.ps1:2185:17
Line |
2185 |                  $authNames[$_auth.Name] = $true
     |                  ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
     | Index operation failed; the array index evaluated to null.
     
I debugged this a bit and noticed that $_auth doesn't have a .Name field for Windows authentication.

What am I missing?

**Comment by Badgerati** [comment]
You're correct! The `Name` property does appear to be missing 🙈 

I'll create a bug for this, and get it fixed 😃
